lib/custom_a11y_rules.js

```diff
@@ -156,6 +171,26 @@ var customRules = {
             evaluate: function(node, options) {
                 return axe.utils.isHidden(node);
             },
-        }
+        },
+        {
+            'id': 'section-heading-first-child',
+            'metadata': {
+                'impact': 'minor',
```

Comment:
I was talking to Christina about this briefly this morning, but wanted to reach out to the group. This entire rule isn't enforced by WCAG (it's a "should" not a "must"), but it is good practice. Adding this test will likely cause failures in other tests where this scenario doesn't match. Having developers fix up the markup would be good.